src/locale/locales/nl/messages.po

```diff
@@ -0,0 +1,7903 @@
+msgid "Clear all storage data (restart after this)"
+msgstr "Wis alle opslaggegevens (hierna opnieuw opstarten)"
+
+#: src/components/forms/SearchInput.tsx:70
+msgid "Clear search query"
+msgstr "Wis zoekopdracht"
+
+#: src/view/screens/Settings/index.tsx:878
+msgid "Clears all storage data"
+msgstr "Wist alle opslaggegevens"
+
+#: src/view/com/modals/DeleteAccount.tsx:216
+msgid "Click here for more information."
+msgstr "Klik hier voor meer informatie."
+
+#: src/view/com/modals/DeleteAccount.tsx:208
+msgid "Click here for more information on deactivating your account"
+msgstr "Klik hier voor meer informatie over het deactiveren van uw account"
+
+#: src/components/TagMenu/index.web.tsx:152
+msgid "Click here to open tag menu for {tag}"
+msgstr "Klik hier om het tagmenu voor {tag} te openen"
+
+#: src/components/dialogs/PostInteractionSettingsDialog.tsx:304
+msgid "Click to disable quote posts of this post."
+msgstr "Klik om citaatberichten voor dit bericht uit te schakelen."
+
+#: src/components/dialogs/PostInteractionSettingsDialog.tsx:305
+msgid "Click to enable quote posts of this post."
+msgstr "Klik om citaatberichten voor dit bericht in te schakelen."
+
+#: src/components/dms/MessageItem.tsx:240
+msgid "Click to retry failed message"
+msgstr "Klik om mislukt privébericht opnieuw te proberen"
+
+#: src/screens/Onboarding/index.tsx:32
+msgid "Climate"
+msgstr "Klimaat"
+
+#: src/components/dms/ChatEmptyPill.tsx:39
+msgid "Clip 🐴 clop 🐴"
+msgstr "Clip 🐴 clop 🐴"
+
+#: src/components/dialogs/GifSelect.tsx:282
+#: src/components/dialogs/VerifyEmailDialog.tsx:246
+#: src/components/dms/dialogs/SearchablePeopleList.tsx:263
+#: src/components/NewskieDialog.tsx:146
+#: src/components/NewskieDialog.tsx:153
+#: src/components/StarterPack/Wizard/WizardEditListDialog.tsx:117
+#: src/components/StarterPack/Wizard/WizardEditListDialog.tsx:123
+#: src/view/com/modals/ChangePassword.tsx:268
+#: src/view/com/modals/ChangePassword.tsx:271
+#: src/view/com/util/post-embeds/GifEmbed.tsx:197
+msgid "Close"
+msgstr "Sluiten"
+
+#: src/components/dialogs/GifSelect.tsx:169
+msgid "Close GIF dialog"
+msgstr "Sluit GIF-dialoogvenster"
+
+#: src/components/Dialog/index.web.tsx:110
+#: src/components/Dialog/index.web.tsx:256
+msgid "Close active dialog"
+msgstr "Sluit actief dialoogvenster"
+
+#: src/screens/Login/PasswordUpdatedForm.tsx:32
+msgid "Close alert"
+msgstr "Sluit waarschuwing"
+
+#: src/view/com/util/BottomSheetCustomBackdrop.tsx:36
+msgid "Close bottom drawer"
+msgstr "Sluit onderste lade"
+
+#: src/components/dialogs/GifSelect.tsx:276
+msgid "Close dialog"
+msgstr "Sluit dialoogvenster"
+
+#: src/view/com/lightbox/ImageViewing/components/ImageDefaultHeader.tsx:36
+msgid "Close image"
+msgstr "Sluit afbeelding"
+
+#: src/view/com/lightbox/Lightbox.web.tsx:129
+msgid "Close image viewer"
+msgstr "Sluit afbeeldingsviewer"
+
+#: src/view/shell/index.web.tsx:67
+msgid "Close navigation footer"
+msgstr "Sluit navigatievoettekst"
```

Comment:
It's not a footer as in text, it's a footer as in the bottom bar navigation.

https://github.com/bluesky-social/social-app/blob/7d34f6bb5c47724b01336eaced7909426be1bdda/src/view/shell/index.web.tsx#L68

```suggestion
msgstr "Sluit onderste navigatiebalk"
```

Is probably most correct? Would match the hint:

```
#: src/view/shell/index.web.tsx:68
msgid "Closes bottom navigation bar"
msgstr "Sluit onderste navigatiebalk"
```

Reply:
Applied suggestion. Thx.